aquadoggo/src/graphql/client/mutation.rs

```diff
@@ -110,21 +124,34 @@ mod tests {
             }
         }"#;
 
-    #[tokio::test]
-    async fn publish_entry() {
-        let (tx, _rx) = broadcast::channel(16);
-        let store = initialize_store().await;
-        let context = HttpServiceContext::new(store, tx);
+    const UPDATE_OPERATION_NO_PREVIOUS_OPS: &str = "A466616374696F6E6675706461746566736368656D617849636861745F30303230633635353637616533376566656132393365333461396337643133663866326266323364626463336235633762396162343632393331313163343866633738626776657273696F6E01666669656C6473A1676D657373616765A26474797065637374726576616C7565764F68682C206D79206669727374206D65737361676521";
 
+    const CREATE_OPERATION_WITH_PREVIOUS_OPS: &str = "A566616374696F6E6663726561746566736368656D617849636861745F30303230633635353637616533376566656132393365333461396337643133663866326266323364626463336235633762396162343632393331313163343866633738626776657273696F6E017370726576696F75735F6F7065726174696F6E738178443030323036356637346636666438316562316261653139656230643864636531343566616136613536643762343037366437666261343338353431303630396232626165666669656C6473A1676D657373616765A26474797065637374726576616C75657357686963682049206E6F77207570646174652E";
+
+    const DELETE_OPERATION_NO_PREVIOUS_OPS: &str = "A366616374696F6E6664656C65746566736368656D617849636861745F30303230633635353637616533376566656132393365333461396337643133663866326266323364626463336235633762396162343632393331313163343866633738626776657273696F6E01";
```

Comment:
I'm not really sure where these should live :raised_eyebrow:

Reply:
If they are so special then its fine where they are. Slightly unrelated: One could also use the cbor macro to generate CBOR data from some JSON-like interface. Then its easy to change the data + you can see what is inside of it.

Reply:
> One could also use the cbor macro

Oh yeh, that'd be better :+1:

Reply:
I had a go at this and got into trouble defining them as `const` (cos of having to make fn calls).... Maybe not worth holding up the PR for slightly nicer const strings :rofl:

Reply:
Sure, its fine as it is :smiley_cat: